cookiecutter.json

```diff
@@ -3,5 +3,6 @@
   "package_name": "{{ cookiecutter.project_name|lower()|replace(' ', '_')|replace('-', '_')|trim() }}",
   "package_slug": "{{ cookiecutter.package_name|replace('_', '-') }}",
   "short_project_description": "",
+  "authors": "{{ cookiecutter.authors | default('') }}",
```

Comment:
I'm pretty sure you can just leave it as `"authors": ""`, but this works just fine